suite-common/wallet-core/src/transactions/transactionsSelectors.ts

```diff
@@ -0,0 +1,107 @@
+import { pipe, A, F } from '@mobily/ts-belt';
+import { createSelector } from '@reduxjs/toolkit';
+
+import { AccountKey } from '@suite-common/suite-types';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { Account, WalletAccountTransaction } from '@suite-common/wallet-types';
+import { enhanceTransaction } from '@suite-common/wallet-utils';
+import { AccountTransaction } from '@trezor/connect';
+
+import { selectAccountByKey } from '../accounts/accountsReducer';
+import type { TransactionsRootState, Txid } from './transactionsReducer';
+
+export const selectIsLoadingTransactions = (state: TransactionsRootState) =>
+    state.wallet.transactions.isLoading;
+
+export const selectTransactionsPerCoin = (state: TransactionsRootState) =>
+    state.wallet.transactions.transactions;
+
+export const selectAccountTxids = (state: TransactionsRootState, accountKey: AccountKey) =>
+    state.wallet.transactions.accountsTransactions[accountKey].allTxids;
+
+export const selectAccountPageTxids = (
+    state: TransactionsRootState,
+    accountKey: AccountKey,
+    page: number,
+) => state.wallet.transactions.accountsTransactions[accountKey].pages[page];
+
+const filterByTxIdAndEchnanceTransactions = (
+    transactions: AccountTransaction[],
+    txidsToKeep: Txid[],
+    account: Account,
+) =>
+    pipe(
+        transactions,
+        A.filter(tx => txidsToKeep.includes(tx.txid)),
```

Comment:
Using function from `mobily/ts-belt` could be handy because according to their benchmarks it should be quite a lot faster than "native" JS equivalent (not sure how they do it but I tried to benchmark myself somehow and it's at least 50% faster everytime)